README.md

```diff
@@ -48,14 +50,32 @@ For a more streamlined workflow, consider using the built-in [app](https://hatch
 
 ## Building
 
-First configure your [project](#project), then select the directory in which to build the executable with the `--root` option and run:
+Before building your application, you must [configure](#configuration) your [project](#project) at the very least.
+
+After you have done that, there are 2 ways to build your application.
+
+### Installation
+
+Select the directory in which to build the executable with the `--root` option and run:
 
 ```
 cargo install pyapp --force --root <DIR>
 ```
 
 The executable will be located at `<DIR>/bin/pyapp.exe` if on Windows or `<DIR>/bin/pyapp` otherwise.
 
+### Local repository
+
+Clone this repository then enter the cloned directory and run:
+
+```
+cargo build --release
+```
+
+The executable will be located at `target/release/pyapp.exe` if on Windows or `target/release/pyapp` otherwise. If a particular [target](https://doc.rust-lang.org/cargo/reference/config.html#buildtarget) has been set then the `release` directory will be nested one level deeper under `target/<TARGET>`.
```

Comment:
This is not correct, when cargo is invoked with `--target` it unconditionally puts it in `target/<triplet/...`, and cross will always specify the target even if you call `cross build` without a specific target.

Reply:
So I should change it to say "if a target has been set or if cross is used"?

Reply:
Ah right, you should be getting a warning though if your default on windows is msvc

```
[cross] warning: `cross` does not provide a Docker image for target x86_64-pc-windows-msvc, specify a custom image in `Cross.toml`.
[cross] note: Falling back to `cargo` on the host.
```

Reply:
Fixed, thank you! https://github.com/ofek/pyapp/commit/1384865c7bb03d9d4fdd598ce7aca5a57c73bd63